README.md

```diff
@@ -25,62 +25,63 @@ This repo has an `about.html` file inside of it, as well as a `styles.css` file.
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
     <title>CSS Intro</title>
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
-
     <h1>Your Name Here</h1>
     <div>
-      <p>This is a bunch of information about myself. I'm from here and there and discovered my
-      love of programming when this happend. When I'm not working I'm busy doing this and that.
-    </p>
+      <p>
+        This is a bunch of information about myself. I'm from here and there and discovered my love
+        of programming when this happened. When I'm not working I'm busy doing this and that.
+      </p>
     </div>
 
-    <img src="https://www.breakthrough-pt.com/wp-content/uploads/2014/11/female-default-profile-photo.png" alt="" >
+    <img
+      src="https://www.breakthrough-pt.com/wp-content/uploads/2014/11/female-default-profile-photo.png"
+      alt=""
+    />
 
-    <div >
-      <h3>Here are some of my Skills!</h3>
+    <div>
+      <h2>Here are some of my Skills!</h2>
+        
+      <h3>Languages</h3>
```

Comment:
Nice catch! Great example of how people can improve a11y even when they're just starting out with HTML!